### PR TITLE
Feature/fix color states

### PR DESCRIPTION
**Pull request by patrickliu-sa**
To align new brand portal blue shades in this PR, starting with button component

style-guide-button

### CODE REVIEW

File: react/Button/Button.less [hunk 1]
```diff
@@ -21,21 +21,35 @@
...
 .root {
-  .touchableText(@waving-type-scale);
+  .touchableText(@waving-type-scale, 1px);
   display: inline-block;
   text-decoration: none;
   user-select: none;
   text-align: center;
-  padding: 0 @grid-base * 4;
-  border: 0;
+  border: 1px solid transparent;
```
Comment (mengtzu) [inline react/Button/Button.less]: What's the transparent border achieving here?

File: react/Button/Button.less [hunk 2]
```diff
@@ -97,22 +146,36 @@
...
-  .buttonColor(@actionTurquoise, @saWhite);
+.jobStreet.root_tertiary {
+  .buttonColor(@jsHirerGreen, @saWhite);
+  border-color: currentColor;
 
-  .hoverState({background-color: @actionTurquoise; color: @saWhite;});
-  border: 1px solid @actionTurquoise;
+  .hoverState({background-color: fade(@jsHirerGreen, 10%); color: @jsHirerGreen; border-color: currentColor;});
+  border: 1px solid @jsHirerGreen;
   padding-bottom: 10px;
```
Comment (mengtzu) [inline react/Button/Button.less]: This looks like it's accounting for the border width, might be good to do it with math.  Not strictly necessary for this PR though.

**Comment by mengtzu** [comment]
Looking good, just a couple of minor questions/observations 👍

File: react/Button/Button.less (hunk 1 above)
Comment (patrickliu-sa) [reply react/Button/Button.less]: To maintain **same text horizontal level** precisely for bordered / border-less button, I've put every button will border color and filling them with button color for border-less button.

Comment (mengtzu) [reply react/Button/Button.less]: 👍

**Pull request merged**